### PR TITLE
chore(doc): add docusaurus

### PR DESCRIPTION
**Pull request by kunyan**
## Screenshot(s)

Hello from SPAship   SPAship (2)
Hello from SPAship   SPAship (3)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: docusaurus/docusaurus.config.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+module.exports = {
+  title: "SPAship",
+  tagline: "SPAship is an open source platform for deploying, integrating, and managing single-page apps (SPAs).",
+  url: "https://spaship.com",
+  baseUrl: "/",
+  favicon: "img/favicon.ico",
+  organizationName: "Red Hat", // Usually your GitHub org/user name.
```
Comment (sayak-sarkar) [inline docusaurus/docusaurus.config.js:7]: @kunyan The Git Org name for SPAship is SPAship.
```suggestion
  organizationName: "SPAship", // Usually your GitHub org/user name.
```

File: docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  projectName: "spaship/spaship", // Usually your repo name.
```
Comment (sayak-sarkar) [inline docusaurus/docusaurus.config.js:8]:
```suggestion
  projectName: "spaship", // Usually your repo name.
```

File: docusaurus/docusaurus.config.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        {
+          title: "Community",
+          items: [
+            {
+              label: "Stack Overflow",
+              href: "https://stackoverflow.com/questions/tagged/spaship",
+            },
+            {
+              label: "Discord",
+              href: "https://discordapp.com/invite/spaship",
```
Comment (sayak-sarkar) [inline docusaurus/docusaurus.config.js:61]: This link wasn't working so created a new invite link for a SPAship Discord room.
```suggestion
              href: "https://discord.gg/QVznWBk",
```

File: docusaurus/docusaurus.config.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            },
+            {
+              label: "Twitter",
+              href: "https://twitter.com/spaship",
```
Comment (sayak-sarkar) [inline docusaurus/docusaurus.config.js:65]: Do we even have this account? Twitter says that this account has been suspended.

Comment (kunyan) [reply docusaurus/docusaurus.config.js]: Removed

**Pull request merged**